app/views/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<script src="https://cdn.auth0.com/js/lock/10.7.2/lock.min.js"></script>
```

Comment:
Should maybe upgrade Lock while you're in here?

https://github.com/auth0/lock#install